index.html

```diff
@@ -10,5 +10,13 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech-div">
+    <span class="tech-span">
```

Comment:
What is this for? `<span>` is used for text, `div` is for container. You should remove this span, div will contain the tech